src/os/ui/menu/listmenu.js

```diff
@@ -193,6 +215,46 @@ os.ui.menu.list.onSortSelected = function(event) {
 };
 
 
+/**
+ * Handle the "Color Selected" menu event.
+ *
+ * @param {os.ui.menu.MenuEvent} event The menu event.
+ */
+os.ui.menu.list.onColorSelected = function(event) {
+  var context = event.getContext();
+  if (os.instanceOf(context, os.source.Vector.NAME)) {
+    var source = /** @type {!os.source.Vector} */ (context);
+    if (source && source.getSelectedItems()) {
+      var items = source.getSelectedItems();
+
+      // call the confirm window from this context so it doesn't appear in the wrong tab
+      os.ui.window.launchConfirmColor(function(color) {
+        source.setColor(items, os.style.toRgbaString(color));
```

Comment:
@jsalankey Calling toRgbaString() here, outside the loop, instead of toHexString() in setColor() inside the loop.  Ty ty.